### ISSUE TITLE
Accessibility – Subpart titles in the TOC don't have sufficient color contrast

### ISSUE THREAD

**Issue by carodew** [18F/atf-eregs#514]
<img width="559" alt="screen shot 2017-08-08 at 3 16 55 pm" src="https://user-images.githubusercontent.com/10144074/29090168-b12d5e34-7c4c-11e7-97d6-4d74165bd4b5.png">

"This element has insufficient contrast at this conformance level (WCAG 2.0 AA). Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.8:1."

**Comment by cmc333333** [comment]
This should be resolved once eregs/regulations-site#505 makes its way down.
